hw/drivers/sensors/lis2dw12/include/lis2dw12/lis2dw12.h

```diff
@@ -89,6 +89,38 @@ extern "C" {
 #define LIS2DW12_INT_SRC_WU_IA                  0x02
 #define LIS2DW12_INT_SRC_FF_IA                  0x01
 
+#define LIS2DW12_WAKE_UP_SRC_FF_IA              0x20
+#define LIS2DW12_WAKE_UP_SRC_SLEEP_STATE_IA     0x10
+#define LIS2DW12_WAKE_UP_SRC_WU_IA              0x08
+#define LIS2DW12_WAKE_UP_SRC_X_WU               0x04
+#define LIS2DW12_WAKE_UP_SRC_Y_WU               0x02
+#define LIS2DW12_WAKE_UP_SRC_Z_WU               0x01
+
+#define LIS2DW12_TAP_SRC_TAP_IA                 0x40
+#define LIS2DW12_TAP_SRC_SINGLE_TAP             0x20
+#define LIS2DW12_TAP_SRC_DOUBLE_TAP             0x10
+#define LIS2DW12_TAP_SRC_TAP_SIGN               0x08
+#define LIS2DW12_TAP_SRC_X_TAP                  0x04
+#define LIS2DW12_TAP_SRC_Y_TAP                  0x02
+#define LIS2DW12_TAP_SRC_Z_TAP                  0x01
+
+#define LIS2DW12_SIXD_SRC_6D_IA                 0x40
```

Comment:
You use 6d here, and sixd and one or two other places, but everywhere else. My datasheet references 6d so I dont hate that everywhere personally.

Reply:
I feel this is fine. I see that SIXD is not getting used anywhere. I see that Mathew I using 6d everywhere.